Spectrum/src/com/unhurdle/spectrum/TextField.as

```diff
@@ -228,7 +228,11 @@ package com.unhurdle.spectrum
       }
       invalidTooltip = new AdaptiveTooltipBead();
       invalidTooltip.flavor = "negative";
-      addBead(invalidTooltip);
+      invalidIcon.addBead(invalidTooltip);
+      invalidIcon.setStyle("pointer-events","auto");
+      COMPILE::JS{
+            invalidIcon.width = invalidIcon.element.clientWidth;
```

Comment:
Why is this necessary?

Reply:
For the position of the tooltip. determinePosition() returns pt.x =NaN (=0) because the width of the icon is undefined,